cpp/src/copying/contiguous_split.cu

```diff
@@ -63,7 +63,7 @@ struct src_buf_info {
                int _offset_stack_pos,
                int _parent_offsets_index,
                bool _is_validity,
-               int _column_offset)
+               size_type _column_offset)
```

Comment:
It doesn't seem like `_column_offset` value is used as part of this change. What was the reason for the change?

Reply:
Just making it consistent with the declaration of the parameter in the struct itself.

`size_type column_offset;   // offset in the case of a sliced column`

Reply:
Just making it consistent with the declaration of the parameter in the struct itself.

size_type column_offset; // offset in the case of a sliced column